skycoin/skywire
# Visor UI

Did you run `make format && make check`? Yes

Fixes #352

Changes:	
- Merge hypervisor into visor
- Remove hypervisor binary
- Implement requests from https://github.com/SkycoinProject/skywire-mainnet/issues/446#issuecomment-662631885

How to test this PR:
- Checkout `skywire-services` against https://github.com/SkycoinPro/skywire-services/pull/216
- Build and run the integration env
- Go to https://localhost:8000 and check if the UI works
- Check if the `GET api/visors/${nodeKey}/summary` endpoint works
